WordPress/src/test/java/org/wordpress/android/ui/sitecreation/SiteCreationMainVMTest.kt

```diff
@@ -249,6 +255,19 @@ class SiteCreationMainVMTest {
         verify(wizardManager).setCurrentStepIndex(index)
     }
 
+    @Test
+    fun oldSiteCreationDataClearedWhenReturningToPreviousStep() {
+        // See issue #10189 - unintended data retained if user goes backwards in wizard
+        whenever(wizardManager.showNextStep()).then {
```

Comment:
I believe the `whenever` is not needed here. Imho it makes the test a bit more difficult to understand. Wdyt?

Reply:
Yeah, I think we can get away with taking that out here, since it is documented in previous tests that we need showNextStep() to not default to SEGMENT when we test setting segmentID, but it is okay for testing other values.

`@Test
    fun siteCreationStateUpdatedWithSelectedSegment() {
        whenever(wizardManager.showNextStep()).then {
            wizardManagerNavigatorLiveData.value = SiteCreationStep.VERTICALS
            Unit
        }
        viewModel.onSegmentSelected(SEGMENT_ID)
        assertThat(currentWizardState(viewModel).segmentId).isEqualTo(SEGMENT_ID)
    }
`

   ` @Test
    fun siteCreationStateUpdatedWithSelectedVertical() {
        viewModel.onVerticalsScreenFinished(VERTICAL_ID)
        assertThat(currentWizardState(viewModel).verticalId).isEqualTo(VERTICAL_ID)
    }
`

I'll remove that whenever so the test is easier to read.